### PR TITLE
Feature/update readme

### PR DESCRIPTION
**Pull request by Yekai-Yu**
Preview at
https://github.com/lulucikyo/CheXNet/tree/feature/update-readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,81 @@
...
-- train_val_sample10k.txt - contains 10k images from labeled_train_val_list.txt
\ No newline at end of file
+[CheXNet](https://stanfordmlgroup.github.io/projects/chexnet/) Replication and Improvment experiments for *CS 598 Deep Learning for Healthcare*
+
+This project took [ChestX-ray 14](https://openaccess.thecvf.com/content_cvpr_2017/papers/Wang_ChestX-ray8_Hospital-Scale_Chest_CVPR_2017_paper.pdf) image dataset and predicted probabilities for 14 types of chest diesease.
+
+## Dataset
+The ChestX-ray 14 dataset contains 112,120 chest X-ray images of 30,805 unique patients with 14 disease labels. As per the original work, we roughly split the dataset into training set (70%), validation set (10%) and test set (20%), with no patient overlaps between dataset partitions. 
+
+## Directory Structure
```
Comment (lulucikyo) [inline README.md:9]: Data_Entry_2017_v2020.csv is not listed.
The name of the python file has changed when clean up.
Better explain that our default python file is the replication of CheXNet

File: README.md [hunk 2]
```diff
@@ -1,13 +1,81 @@
...
+
+final_test.txt - test set: filename, label vector
+
+final_train.txt - train set: filename, label vector
+
+final_val.txt - validation set: filename, label vector
+
+***replication_chexnet_cuda_local\*.py - the main program***
+
+## Experiment designs
```
Comment (lulucikyo) [inline README.md]: Maybe add a paragraph explaining our effort of variant experiments.

File: README.md [hunk 3]
```diff
@@ -1,13 +1,81 @@
...
+Pneumonia |	0.633 |	0.713 |	0.768 |	0.7414
+Pneumothorax |	0.806 |	0.841 |	0.8887 |	0.8886
+Consolidation |	0.708 |	0.788 |	0.7901 |	0.8269
+Edema |	0.835 |	0.882 |	0.8878 |	0.8848
+Emphysema |	0.815 |	0.829 |	0.9371 |	0.9336
+Fibrosis |	0.769 |	0.767 |	0.8047 |	0.8194
+Pleural_Thickening |	0.708 |	0.765 |	0.8062 |	0.8115
+Hernia |	0.767 |	0.914 |	0.9164 |	0.9198
+Average AUROC |	0.738 |	0.803 |	0.8414 |	0.8446
+## Prerequisite
```
Comment (lulucikyo) [inline README.md:63]: cuda

File: README.md [hunk 4]
```diff
@@ -1,13 +1,81 @@
...
+6. Specify image folder path, i.e.
+```
+DATA_PATH = './images_converted256/'
+```
+7. Run
+```
+python replication_chexnet_cuda_local.py
+```
+## Contributor
+[Xi Li](https://github.com/lulucikyo), [Yu Liu](https://github.com/lytinahome), [Liping Xie](https://github.com/lipingxie), [Yekai Yu](https://github.com/Yekai-Yu)
```
Comment (lulucikyo) [inline README.md:91]: Also put email?

File: README.md (hunk 2 above)
Comment (Yekai-Yu) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (Yekai-Yu) [reply README.md:9]: Data_Entry_2017_v2020.csv is part of the original dataset download assets, removed it from our repo, since it's out of the scope

Comment (Yekai-Yu) [reply README.md:9]: other comments done

File: README.md (hunk 3 above)
Comment (Yekai-Yu) [reply README.md:63]: done

**Review by lulucikyo — APPROVED** [review]

**Pull request merged**